packages/ember-htmlbars/tests/helpers/with_test.js

```diff
@@ -1,262 +1,13 @@
-    foo: {
-      isTruthy: true
-    }
-  });
-  runAppend(view);
-
-  equal(view.$().text(), 'True');
-
-  run(function() {
-    set(view, 'foo.isTruthy', false);
-  });
-
-  equal(view.$().text(), 'False');
-
-  run(function() {
-    set(view, 'foo.isTruthy', true);
-  });
-
-  equal(view.$().text(), 'True');
-});
-
-QUnit.test('should respect `isTruthy` field on the context object', function() {
-  view = EmberView.create({
-    template: compile('{{#with foo}}True{{else}}False{{/with}}'),
-    context: {
-      foo: {
-        isTruthy: true
-      }
-    }
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'True');
-
-  run(function() {
-    set(view, 'context.foo.isTruthy', false);
-  });
-
-  equal(view.$().text(), 'False');
-
-  run(function() {
-    set(view, 'context.foo.isTruthy', true);
-  });
-
-  equal(view.$().text(), 'True');
-});
-
-QUnit.test('the scoped variable is not available outside the {{with}} block.', function() {
-  view = EmberView.create({
-    template: compile('{{name}}-{{#with other as |name|}}{{name}}{{/with}}-{{name}}'),
-    context: {
-      name: 'Stef',
-      other: 'Yehuda'
-    }
-  });
-
-  runAppend(view);
-  equal(view.$().text(), 'Stef-Yehuda-Stef', 'should be properly scoped after updating');
-});
-
-QUnit.test('nested {{with}} blocks shadow the outer scoped variable properly.', function() {
-  view = EmberView.create({
-    template: compile('{{#with first as |ring|}}{{ring}}-{{#with fifth as |ring|}}{{ring}}-{{#with ninth as |ring|}}{{ring}}-{{/with}}{{ring}}-{{/with}}{{ring}}{{/with}}'),
-    context: {
-      first: 'Limbo',
-      fifth: 'Wrath',
-      ninth: 'Treachery'
-    }
-  });
-
-  runAppend(view);
-  equal(view.$().text(), 'Limbo-Wrath-Treachery-Wrath-Limbo', 'should be properly scoped after updating');
-});
-
-QUnit.module('Handlebars {{#with keyword as |foo|}}');
-
-QUnit.test('it should support #with view as |foo|', function() {
-  var view = EmberView.create({
-    template: compile('{{#with view as |myView|}}{{myView.name}}{{/with}}'),
-    name: 'Sonics'
-  });
-
-  runAppend(view);
-  equal(view.$().text(), 'Sonics', 'should be properly scoped');
-
-  run(function() {
-    set(view, 'name', 'Thunder');
-  });
-
-  equal(view.$().text(), 'Thunder', 'should update');
-
-  runDestroy(view);
-});
-
-QUnit.test('it should support #with name as |foo|, then #with foo as |bar|', function() {
-  var view = EmberView.create({
-    template: compile('{{#with name as |foo|}}{{#with foo as |bar|}}{{bar}}{{/with}}{{/with}}'),
-    context: { name: 'caterpillar' }
-  });
-
-  runAppend(view);
-  equal(view.$().text(), 'caterpillar', 'should be properly scoped');
-
-  run(function() {
-    set(view, 'context.name', 'butterfly');
-  });
-
-  equal(view.$().text(), 'butterfly', 'should update');
-
-  runDestroy(view);
-});
-
 QUnit.module('Handlebars {{#with this as |foo|}}');
 
-QUnit.test('it should support #with this as |qux|', function() {
```

Comment:
Actually, I can't find that test anymore, I suppose we lost it when merging multiple cases together? I see that we have the shared test case for `this`, but this is testing `{{#with this as  |foo|}}`. Can you link me to the new test or add it back?